core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -1516,6 +1516,13 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.ENFORCE)
           .setScope(Scope.MASTER)
           .build();
+  public static final PropertyKey MASTER_METASTORE_BLOCK_ITERATOR_READAHEAD_SIZE =
+      new Builder(Name.MASTER_METASTORE_BLOCK_ITERATOR_READAHEAD_SIZE)
+          .setDefaultValue("10000")
```

Comment:
Should we make this larger?

Reply:
I was under impression that it was count, but turns out it's bytes. Made it 4MB to start with.